src/app/api/detailLife.graphql

```diff
@@ -0,0 +1,12 @@
+query detailLife($lifeID: ObjectID!) {
+    getLife(id: $lifeID){
+        id
+        firstName
+        lastName
+        fullName
+        description
+        birthday
+        hobbies
+        title
```

Comment:
You can create a graphql `fragment` so you can re-use the content for both `getLife` and `getLives`